docs/development/maintenance.md

```diff
@@ -1,39 +1,92 @@
-ILIAS Maintenance
-=================
-The development of the ILIAS source code is coordinated and maintained by a coordination team within the ILIAS network. Besides the main responsibilities for the project, several developers and users are maintaining certain modules of ILIAS.
-
-# Special Roles
-
-* **Product Management**: [Matthias Kunkel]
-* **Technical Board**: [Alexander Killing], [Michael Jansen], [Timon Amstutz], [Richard Klees], [Stephan Winiker]
-* **Testcase Management**: [Fabian Kruse]
-* **Documentation**: [Florian Suittenpointner]
-* **Online Help**: [Alexandra Tödt]
-
-# Maintainers
-We highly appreciate to get new developers but we have to guarantee the sustainability and the quality of the ILIAS source code. The system is complex for new developers and they need to know the concepts of ILIAS that are described in the development guide.
- 
-Communication among developers that are working on a specific component needs to be assured. Final decision about getting write access to the ILIAS development system (Github) is handled by the product manager.
- 
-ILIAS is currently maintained by three types of Maintainerships:
-
-- First Component Maintainer
-- Second Component Maintainer
-- [Coordinator Model](maintenance-coordinator.md) 
- 
-The following rules must be respected for everyone involved in the programming of ILIAS for all components having a listed component maintainer (see bellow):
-
-1. Decisions on new features or feature removals are made by the responsible first maintainer and the product manager in the Jour Fixe meetings after an open discussion.
-2. All components have a first and second maintainer. Code changes are usually done by the first maintainer. The first maintainer may forward new implementations to the second maintainer.
-
-Responsibilities of a component maintainer:
-
-- Component maintainer must assure maintenance of their component for at least three years (approx. three ILIAS major releases).
-- Component maintainers must agree to coordinate the development of their component with the product manager.
-- Component maintainer are responsible for bug fixing of their component and get assigned related bugs automatically by the [Issue-Tracker](http://mantis.ilias.de).
-
-
-The code base is deviced in several components:
+# Coordinator Model
```

Comment:
```suggestion
# Maintenance
```

Reply:
Hab das noch zu "Maintainer Model" geändert um das parallel zu dem Coordinator Model zu halten. Das hängt wahrscheinlich auch mit den Dateinamen zusammen. Ich würde das ebenfalls zur Diskussion in das Forum stellen.